packages/core-ethereum/crates/core-ethereum-actions/src/environment.rs

```diff
@@ -314,16 +314,63 @@ impl ResolvedNetwork {
     }
 }
 
+#[derive(Serialize, Clone)]
+#[cfg_attr(feature = "wasm", wasm_bindgen::prelude::wasm_bindgen(getter_with_clone))]
+pub struct DeploymentExtract {
```

Comment:
Should be probably called `SmartContractConfig`

Reply:
Yeah, was attemtping to stay true to naming to not mix everything up.